backend/src/routes/BankingRoutes.ts

```diff
@@ -71,18 +73,54 @@ async function get(_: IReq, res: IRes) {
 const prettyPrintResponse = (response: any) => {
   console.log(util.inspect(response.data, { colors: true, depth: 4 }))
 }
-// Send the Public Token to Plaid in exchange for an "Access Token"
 
+// Send the Public Token to Plaid in exchange for an "Access Token"
 async function put(req: IReq<IPublicToken>, res: IRes) {
   const tokenResponse = await client.itemPublicTokenExchange({
     public_token: req.body.publicToken,
   })
 
   prettyPrintResponse(tokenResponse)
-  ACCESS_TOKEN = tokenResponse.data.access_token
+  ACCESS_TOKEN = tokenResponse.data.access_token as string;
+  
+  database.serialize(() => {
+    database.run('DELETE FROM accessToken', function(err:Error) {
+      if (err) {
+          console.error('Error deleting rows:', err.message);
+      } else {
+          console.log('Rows deleted successfully');
+      }
+  });
+    database.run(
+        'INSERT INTO accessToken (accesstoken) VALUES (?)',
+        [ACCESS_TOKEN],
+        (err: Error) => {
+            if (err) {
+                // Handle error
+                console.log('Error occurred:', err.message);
+            } else {
+                // Insertion successful
+                console.log('AccessToken inserted successfully');
+               
+            }
+        }
+    );
+    const sqlQuery = 'SELECT * FROM accessToken';
+    database.all<any>(sqlQuery, (err: Error | null, rows: any[]) => {
```

Comment:
Try to give this a type so you can replace the `<any>` and `any[]`. I think this will require you to create a new class or interface that represents the output of the query.